src/api/layers/ExternalWMTSLayer.class.js

```diff
@@ -18,11 +18,11 @@ export default class ExternalWMTSLayer extends ExternalLayer {
     constructor(name, opacity, visible, getCapabilitiesUrl, externalLayerId, attributions) {
         super(
             // as we are encoding the name in getID(), we must decode it here when receiving it through the URL parser
-            decodeURIComponent(name),
+            name,
             LayerTypes.WMTS,
             externalLayerId,
             // same thing with the getCap URL, we encode it in getID, so it must be decoded
-            decodeURIComponent(getCapabilitiesUrl),
+            getCapabilitiesUrl,
```

Comment:
The decoding is totally removed as we let now vue do the encoding and decoding, we had a hack to prevent encoding of the layers query param and where handling the encoding/decoding in the layer classes but that causes troubles.